### PR TITLE
Update Getting Ready for Class chapter

### PR DESCRIPTION
**Pull request by tatarhy**
This is some improvement of Getting Ready for Class chapter.

### CODE REVIEW

**Comment by beardofedu** [comment]
@tatarhy , can you replace the other reference to Unix on line 55? This is looking good to ship otherwise.

**Comment by ericalloyd** [comment]
Thanks for updating, @tatarhy! After addressing @beardofedu's feedback, this LGTM 🚀

File: docs/01_getting_ready_for_class.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can set up your free account by following these steps:
 
 If you already have an account, verify that you can visit github.com within your organization's network.
 
-GitHub is designed to run on the current versions of all major browsers. In particular, if you use Microsoft's Internet Explorer (IE), you must be using the latest version. Take a look at our list of [supported browsers](https://help.github.com/articles/supported-browsers/).
+GitHub is designed to run on the current versions of all major browsers. Take a look at our list of [supported browsers](https://help.github.com/articles/supported-browsers/).
```
Comment (parkerbxyz) [inline docs/01_getting_ready_for_class.md:23]:
```suggestion
GitHub is designed to run on the current versions of all major browsers. Take a look at our list of [supported browsers](https://docs.github.com/github/getting-started-with-github/supported-browsers).
```

File: docs/01_getting_ready_for_class.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ If your clone is unsuccessful, read about [authenticating with GitHub from Git](
 
 #### Proxy configuration
 
-If your organization uses a proxy, you will need to configure the proxy settings in Git. Open Git Bash (on Windows) or Terminal (on Mac or *nix) and complete the appropriate steps below:
+If your organization uses a proxy, you will need to configure the proxy settings in Git. Open Git Bash (on Windows) or Terminal (on Mac or \*nix) and complete the appropriate steps below:
```
Comment (parkerbxyz) [inline docs/01_getting_ready_for_class.md:80]: May I ask why escaping the asterisk here is necessary? It seems to render fine in [the manual](https://githubtraining.github.io/training-manual/#/01_getting_ready_for_class?id=proxy-configuration) without the `\`.

**Review by parkerbxyz — CHANGES_REQUESTED** [review]
Thanks for your contribution, @tatarhy! I left a suggestion and question for you to consider.

File: docs/01_getting_ready_for_class.md (hunk 2 above)
Comment (beardofedu) [reply docs/01_getting_ready_for_class.md:80]: Just a quick test, the editor shows the line as bold without the `\` which might be why the escape character was included. The preview doesn't show the content as bold, but when writing in an editor it does 🤷 .

Comment (parkerbxyz) [reply docs/01_getting_ready_for_class.md:80]: I see. The `\` properly escapes the `*` in the GitHub editor and I verified that it doesn’t show up on the published web page by serving this branch locally, so this change looks good. 👍

**Comment by tatarhy** [comment]
Thank you for your reviewing and feedbacks!
@ericalloyd The feedback from @beardofedu is addressed by @parkerbxyz commit. And I addressed also @parkerbxyz suggestion.

**Review by beardofedu — APPROVED** [review]

File: docs/01_getting_ready_for_class.md (hunk 1 above)
Comment (parkerbxyz) [reply docs/01_getting_ready_for_class.md]: Thanks, @tatarhy! Just for future reference, you can quickly and easily apply suggested changes right here from the pull request with the click of a button; no need to make the changes manually: https://docs.github.com/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

That might save you some time and effort in the future. 😉

**Review by parkerbxyz — APPROVED** [review]
Thanks again, @tatarhy! ✨

**Pull request merged**